lib/index.js

```diff
@@ -779,10 +789,7 @@ export default {
 	/**
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/index.js line...
  
  
  DangerID: danger-id-Danger;
  File: lib/index.js;
  Line: 789;
-->

- :warning: lib/index.js line 789 – Missing JSDoc for parameter 'callback'. (valid-jsdoc)